index.js

```diff
@@ -70,6 +100,47 @@ async function run() {
       })
     }
 
+    if (environmentSecrets) {
+
+      // If environment array is missing, create it
+      if (!Array.isArray(containerDef.secrets)) {
+        containerDef.secrets = [];
+      }
+
+      // Get pairs by splitting on newlines
+      environmentSecrets.split('\n').forEach(function (line) {
```

Comment:
Nit; this could be turned into a `.map()` that maps each environment secret line to type `{ name: string, valueFrom: string }` and then we could merge these values into `containerDef.secrets` outside of the `forEach` loop.